ambry-api/src/main/java/com.github.ambry/config/HelixAccountServiceConfig.java

```diff
@@ -23,6 +23,14 @@
   public static final String UPDATER_SHUT_DOWN_TIMEOUT_MS_KEY =
       HELIX_ACCOUNT_SERVICE_PREFIX + "updater.shut.down.timeout.ms";
   public static final String BACKUP_DIRECTORY_KEY = HELIX_ACCOUNT_SERVICE_PREFIX + "backup.dir";
+  public static final String INVALID_ZK_CLIENT_CONNECT_STRING = "";
+
+  /**
+   * The ZooKeeper server address. This config is required when using {@code HelixAccountService}.
+   */
+  @Config(HELIX_ACCOUNT_SERVICE_PREFIX + "zk.client.connect.string")
+  @Default(INVALID_ZK_CLIENT_CONNECT_STRING)
```

Comment:
I don't think this needs a default. This class is instantiated only in `HelixAccountServiceFactory` where this value must be valid (wasn't true for the `HelixPropertyStoreConfig`

Reply:
yep this doesn't need a default. You can also define a constant for the config key (`ZK_CLIENT_CONNECT_STRING_KEY`) if it helps.